apps/chat/src/components/Common/SelectFolder/SelectFolderList.tsx

```diff
@@ -19,22 +19,24 @@ import Folder, { FolderProps } from '@/src/components/Folder/Folder';
 
 interface Props<T, P = unknown> {
   folderProps: Omit<FolderProps<T, P>, 'currentFolder'>;
-  handleToggleFolder: (folderId?: string) => void;
+  handleFolderSelect: (folderId?: string) => void;
```

Comment:
`folderId` can be undefined
```suggestion
  handleToggleFolder: (folderId: string) => void;
```

Reply:
It cannot be undefined so removed '?'